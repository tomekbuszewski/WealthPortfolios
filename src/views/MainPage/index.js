import * as React from 'react';

import Section from '@src/components/Section';

const MainPage = () => (
  <Section title="Index">
    <div>Hello, please select something from the menu.</div>
  </Section>
);

export default MainPage;
