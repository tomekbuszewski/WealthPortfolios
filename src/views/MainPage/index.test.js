import * as React from 'react';
import renderer from 'react-test-renderer';

import MainPage from './';

describe ('MainPage tests', () => {
  it ('should match snapshot', () => {
    const component = renderer.create(<MainPage />);
    const tree = component.toJSON();

    expect(tree).toMatchSnapshot();
  });
});
