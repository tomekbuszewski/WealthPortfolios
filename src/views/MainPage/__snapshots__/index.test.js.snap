// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MainPage tests should match snapshot 1`] = `
<section
  className="sc-bdVaJa gPOcxX section hero"
>
  <header
    className="title"
  >
    Index
  </header>
  <div
    className="content is-large"
  >
    <div>
      Hello, please select something from the menu.
    </div>
  </div>
</section>
`;
